sys-auth/yubikey-personalization-gui/Manifest

```diff
@@ -1,2 +1,3 @@
 DIST yubikey-personalization-gui-3.1.23.tar.gz 1450340 BLAKE2B fb183d939334b13faf1464dc65a0a03180a2b402ae628f556dba7211e66e89628718e797a86189f654ca88225c3b834df67b497e7a94d0e210edc433bfff49b8 SHA512 912bcbcac5cad285c43995f646de6aa68cc99a4d047b7632786cd93e38c031e2fb8dcc4f57f45647ba3e276de03aaea6ed9757f72ec2c42a43780c19edcca51b
 DIST yubikey-personalization-gui-3.1.24.tar.gz 947135 BLAKE2B 846fa6792c71b57ffb99c548efe6780571c1d52061393805f0f9a76a7e7722b3fc2851ead913b09028d7798ae7bab4670260cae63a98ef0c0d1b176266fa4338 SHA512 149237a719277a6571916473d31e6099c3f3a2003fcb2893c2becee80788102496dbbc8e424f75fe1f1dd3ac87122bd35b6908cf784458635069d89e7e706cfa
+DIST yubikey-personalization-gui-3.1.25.tar.gz 951145 BLAKE2B f500ef4f591ed48d660503fe9f85e71ff19efc6b6dfef7906684e0167ca5f4f77f007541911ac62efb568022777fd620d0b368fc6f1894fd1883c22f180716ab SHA512 1171bca2caecf998dda57b76a9e11254ee976568f303fa500b01c3d73ec4e0bf39ef71928f1f87daed450f7be137cb90be0cd08aeb9a9ceb0450798de91ae8f2
```

Comment:
```
!!! Fetched file: yubikey-personalization-gui-3.1.25.tar.gz VERIFY FAILED!
!!! Reason: Filesize does not match recorded size
```

Reply:
Dammit, the file linked in the [yubikey site](https://www.yubico.com/support/knowledge-base/categories/articles/yubikey-personalization-tools/) isn't the same they have in github. 
I'll check the differences before pushing a change.

Reply:
The tarball I had included a ChangeLog, the github tarball doesn't. No other changes.